voxpupuli-acceptance.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
 
   # Testing
   s.add_runtime_dependency 'bcrypt_pbkdf', '~> 1.1'
-  s.add_runtime_dependency 'beaker', '>= 4.33', '< 6'
+  s.add_runtime_dependency 'beaker', '>= 4.33', '< 7'
```

Comment:
my preference is `~> 7.0` to ensure we use 7.0 in all of our pipelines and not a wild mix.